tests/instances.tcl

```diff
@@ -499,8 +499,7 @@ while 1 {
             # letting the tests resume, so we'll eventually reach the cleanup and report crashes
 
             if {$::exit_on_failure} {
-                puts -nonewline "(Fast fail: test will exit now)"
-                flush stdout
+                puts "(Fast fail: test will exit now)"
```

Comment:
Why was this change made?

Reply:
I wrote in the PR top comment: "Additional fix: End tests with a linebreak when running with the --fastfail flag. A program that doesn't end its output with a linebreak annoys the prompt in the terminal."

When it ends without a linebreak in a shell, the prompt gets printed on the same line like this:

```
(Fast fail: test will exit now)viktor@fedora:~/repos/valkey$ 
```